src/dosbox.cpp

```diff
@@ -725,9 +728,21 @@ void DOSBOX_Init()
 	secprop->AddInitFunction(&VGA_Init);
 	secprop->AddInitFunction(&KEYBOARD_Init);
 
-
 #if defined(PCI_FUNCTIONALITY_ENABLED)
-	secprop=control->AddSection_prop("pci", &PCI_Init); // PCI bus
+	secprop=control->AddSection_prop("pci", &PCI_Init, false); // PCI bus
+
+#if C_VOODOO
+	secprop->AddInitFunction(&VOODOO_Init, false);
+
+	const char* voodootypes[] = { "12mb", "4mb", "disabled", 0 };
+	Pstring = secprop->Add_string("voodoo", only_at_start, "12mb");
+	Pstring->Set_values(voodootypes);
+	Pstring->Set_help("RAM amount of emulated Vodooo 3dfx card.");
```

Comment:
This will need to be revisited I think because it seems to also toggle whether 1 or 2 texturing units are available.

Voodoo 1 — 4 MB, 1 texturing unit
Voodoo 2 — 8 or 12 MB, 2 texturing units

Reply:
Voodoo 2 SLI - double of everything (e.g. 16MB or 24MB, 4 texturing units, etc.), **1024x768** support
Voodoo 2 Banshee - 8 or 16MB, 1 texturing unit

Reply:
Voodoo 5 - 32 or 64 or 128 MB, 4 or 8 texturing units (mentioning for completeness; I assume that's out of scope...)

Reply:
It should also be clear to the user whether he is enabling the original Voodoo (Voodoo 1), or a Voodoo 2.

Reply:
Do some games only work properly with voodoo 1 (and say 4 MB of RAM) versus later gen games that needed voodoo 2 and 12 MB?

Ideally there's an "everything works perfect" setting and we simply lock it in without any conf settings needed.

Reply:
The code for 12mb triggers codepaths for Voodoo 1 with 2 TMUs; configuration that never existed in reality: https://en.wikipedia.org/wiki/3dfx_Interactive#Products

I think we should drop it and upgrade 4MB to 8MB option to match the best version of Voodoo 1 card.

To be discussed in separate PR.